crates/dslab-mp-python/python-tests/node.py

```diff
@@ -0,0 +1,45 @@
+from dslabmp import Context, Message, Process, StateMember
+from typing import List
+
+class Node(Process):
+    def __init__(self, node_id: str):
+        self.data = StateMember(["elem1", (2, 3)])
+        self._id = StateMember(node_id)
+        
+        # examples of set/get state member
+        self._id = 'new_node_id'
+        self.secret = '''YOU SHOLDN'T SEE IT AFTER RELOAD,
+                         IT IS NOT A STATE MEMBER OF NODE %s''' % self._id
+    
+    def on_local_message(self, msg: Message, ctx: Context):
+        assert self._id == 'new_node_id', 'get/set attributes work bad with state members'
+        tmp_value = "SHOULD BE DROPPED"
+        if self.data == tmp_value:
+            ctx.send_local(msg)
+        
+        # we suppose it will be discarded later
+        self.data = tmp_value
+        assert type(self.__dict__['data']) == StateMember, 'field "data" must be a StateMember even after assignment'
+
+        if self.secret is None:
+           pass
```

Comment:
PEP 8: E111 indentation is not a multiple of 4

Reply:
fixed